src/internal/alloc.rs

```diff
@@ -43,6 +43,10 @@ impl<F> Allocator<F> {
         self.sectors.version()
     }
 
+    pub fn inner(&self) -> &Sectors<F> {
+        &self.sectors
```

Comment:
Sorry, I guess my suggestion before was ambiguous.  My intention was that `inner()` would return the `F` object (similar to `into_inner()`), so this method should look like

```rust
pub fn inner(&self) -> &F {
    self.sectors.inner()
}
```

and similarly for `directory.rs` and `minialloc.rs`.

Reply:
Oh, understood. I'll update